app/src/androidTest/java/com/github/sdp_begreen/begreen/activities/MainActivityTest.kt

```diff
@@ -335,7 +334,7 @@ class MainActivityTest {
     @Test
     fun contactUsMessageIsSentToDatabaseWithAddFeedback() {
         runTest {
-            `when`(db.addFeedback(org.mockito.kotlin.any(), org.mockito.kotlin.any() , org.mockito.kotlin.any(), org.mockito.kotlin.any()))
+            `when`(db.addFeedback(any(), any() , any(), any()))
```

Comment:
Well seen to statically import it, it's a lot cleaner!

Reply:
Yeah, I don't really get why we thought we had to do it like that, seems to work fine now ;)